.gitignore

```
@@ -4,3 +4,4 @@ dist/**/*
 **/__pycache__/**/*
 **/*.egg-info/**/*
 .sass-cache/*
+build
```

```suggestion
/build
```
We don't want it applied to sub folders containing `build`. Btw what creates this folder?

---

Done in [c479e7b](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/87/commits/c479e7bd6089f110ccc0473ff1fe61439152695f)

This gets built when I install a local version of rocm-docs-core for testing via `pip install <dir>/rocm-docs-core`